lib/rails_event_store_mongoid/event.rb

```diff
@@ -13,7 +13,22 @@ class Event
     field :meta, type: Hash, default: {}
     field :data, type: Hash, default: {}
 
+    field :timestamp, type: BSON::Timestamp
+
+    def self.create(**args)
```

Comment:
why to override methods instead of setting a default `field :timestamp, type: BSON::Timestamp, default: -> {  BSON::Timestamp.new(0, 0) }` ?